src/layout.rs

```diff
@@ -5,24 +5,22 @@ use leptos_router::{Outlet, RoutingProgress};
 
 #[component]
 pub fn Layout(is_routing: ReadSignal<bool>) -> impl IntoView {
+  let ui_theme = expect_context::<RwSignal<String>>();
+
   view! {
-    <Suspense>
-      <ErrorBoundary fallback=|_| view! { Error! }>
-        <RoutingProgress is_routing max_time=std::time::Duration::from_millis(250)/>
-        <Stylesheet id="leptos" href="/pkg/lemmy-ui-leptos.css"/>
-        <Link rel="shortcut icon" type_="image/ico" href="/favicon.ico"/>
-        <Meta name="description" content="Lemmy-UI-Leptos."/>
-        <Meta name="viewport" content="viewport-fit=cover"/>
-        <Script src="//cdn.jsdelivr.net/npm/eruda"/>
-        <Script>eruda.init();</Script>
-        <Title text="Brand from env"/>
+    <RoutingProgress is_routing max_time=std::time::Duration::from_millis(250)/>
+    <Stylesheet id="leptos" href="/pkg/lemmy-ui-leptos.css"/>
+    <Link rel="shortcut icon" type_="image/ico" href="/favicon.svg"/>
+    <Meta name="description" content="Lemmy-UI-Leptos."/>
+    <Meta name="viewport" content="viewport-fit=cover"/>
+    // <Script src="//cdn.jsdelivr.net/npm/eruda"/>
```

Comment:
@dessalines what is eruda? I saw it's included in the old UI and the join-lemmy repos when in debug mode, but I don't know what it adds that isn't already handled by browser devtools.